README.md

```diff
@@ -106,6 +106,28 @@ When playing with data via MongoDB Compass, always clean all the collections omi
 
 To perform DB migrations we use `migrate-mongo` tool. To config `migrate-mongo` edit `migrate-mongo-config.js` file. To create migration file run `npm run migrate create <version>` in your terminal. Migration file will be created at server/migrations directory. To run all new migrations run `npm run migrate:up` or `npm run migrate up` in your terminal. To rollback, last migration run `npm run migrate:down` or `npm run migrate down` in your terminal. To see migrations status run `npm run migrate status` in your terminal.
 
+## Third-party licenses
+
+To generate json and markdown files containing third-party licenses you need to
+ install globally: [license-checker](https://github.com/davglass/license-checker#readme) and [npm-license-crawler](https://github.com/mwittig/npm-license-crawler):
+
+ #### 1. Install global packages
+
+```
+npm i license-checker npm-license-crawler -g
+```
+
+#### 2. Go inside the directory
+
+```
+cd eoc
+```
+
+#### 3. Run libs script
+```
+npm run libs -j path/to/json/file [-m path/to/md/file]
```

Comment:
what json file and what md file?

Reply:
md file - markdown file is placed in main project folder. It contains a list of all packages with two links: to their online license file and totheir repo. This file is accessible from our repo.

JSON file -  (In our app case is saved in `server/data` folder.) It contains all packages data that we need to send to the front-end app to display the libraries we used in the project. It is used by controllers: `getLibraries` and `getLicense` that extract specific data from it and send this data to the client.